pkg/datastore/datastore.go

```diff
@@ -171,7 +171,7 @@ type ReadWriteTransaction interface {
 	Reader
 
 	// WriteRelationships takes a list of tuple mutations and applies them to the datastore.
-	WriteRelationships(mutations []*v1.RelationshipUpdate) error
+	WriteRelationships(mutations ...*core.RelationTupleUpdate) error
```

Comment:
This change really didn't improve readability or usability at all. There isn't a single call site in the PR that doesn't call this using a slice with the splat `...` operator. Please revert it.

Reply:
here is my rationale for the change: in the test codebase there is _a lot_ of snippets of code list this:

```go
return rwt.WriteRelationships([]*v1.RelationshipUpdate{{
			Operation:    v1.RelationshipUpdate_OPERATION_CREATE,
			Relationship: relationship,
		}})
```

Granted, that became not so much of a big deal once I introduced `common.WriteTuples` which DRY things even further.

There is already another method in the interface. If you feel strongly for this I can change it.

Reply:
addressed in https://github.com/authzed/spicedb/pull/771/commits/f709966b64c80a46f44ef75cfe0dcfa8ddfb31c0